### PR TITLE
V_1

### PR DESCRIPTION
**Pull request by Dima-Kovalskiy**
https://<Dima-Kovalskiy>.github.io/layout_stars/
https://<Dima-Kovalskiy>.github.io/layout_stars/report/html_report/

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://<Dima-Kovalskiy>.github.io/layout_stars/)
+- [TEST REPORT LINK](https://<Dima-Kovalskiy>.github.io/layout_stars/report/html_report/)
```
Comment (polosanya) [inline readme.md:4]:
```suggestion
- [DEMO LINK](https://Dima-Kovalskiy.github.io/layout_stars/)
- [TEST REPORT LINK](https://Dima-Kovalskiy.github.io/layout_stars/report/html_report/)
```

**Review by polosanya — CHANGES_REQUESTED** [review]
Please add the correct clickable links to the PR description.
Also, make sure that you added all the files from src folder to your commit.
Now only readme file is present on GitHub(

**Comment by Dima-Kovalskiy** [comment]
> https://.github.io/layout_stars/

**Comment by Dima-Kovalskiy** [comment]
https://Dima-Kovalskiy.github.io/layout_stars/
https://Dima-Kovalskiy.github.io/layout_stars/report/html_report/

**Pull request closed without merging**